AttitudeManager/FlightModes/Src/fbwa.cpp

```diff
@@ -0,0 +1,32 @@
+#include "fbwa.hpp"
+#include "CommonDataTypes.hpp"
+#include "AM.hpp"
+#include "config.hpp"
+
+namespace AM {
+    AttitudeManagerInput FBWA::run(const AttitudeManagerInput& input) {
+        
+        AttitudeManagerInput mappedOutputs;
+
+        //Importing max/min limits here
+        float yawMin = config::flightmodes[1].tuningData.controlLimits.yawLimit.min;
+        float yawMax = config::flightmodes[1].tuningData.controlLimits.yawLimit.max;
+
+        float rollMin = config::flightmodes[1].tuningData.controlLimits.rollLimit.min;
+        float rollMax = config::flightmodes[1].tuningData.controlLimits.rollLimit.max;
+        
+        float pitchMin = config::flightmodes[1].tuningData.controlLimits.pitchLimit.min;
+        float pitchMax = config::flightmodes[1].tuningData.controlLimits.pitchLimit.max;
+
+        // Mapping inputs to outputs with ratios of min-max
+        mappedOutputs.pitch = pitchMin + (input.pitch - pitchMin) * (pitchMax - pitchMin) / (AM::AttitudeManager::INPUT_MAX - AM::AttitudeManager::INPUT_MIN);
```

Comment:
This math doesn't quite work out. I'm going to try and figure out how to actually do this to maintain zero centering.

I just tried this with input = 0, min = -30, max = 40

pitch = -30 + 30*(10) / 200 = 270 / 200 = 1.3~

Reply:
I think I can find a way to do it without an IF, but

`mappedOutputs.pitch = input.pitch < 0 ? input.pitch * (pitchMin / INP_MIN) : input.pitch * (pitchMAX / INP_MAX)`